src/const/browser.ts

```diff
@@ -24,7 +24,9 @@ export const EVENTS = {
   MOUSE_LEAVE: "mouseleave",
   LOAD: "load",
   ERROR: "error",
-  CLICK: "click"
+  CLICK: "click",
+  CONTEXT_LOST: "webglcontextlost",
+  CONTEXT_RESTORE: "webglcontextrestored"
```

Comment:
It is okay to omit webgl. But I guess `RESTORED' would be nice?

Reply:
Good point 👍